app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -25,41 +25,9 @@ keywords:
     - Grants
     - Financial Support
     - Undergraduate
-    - Troops to Teachers
-    - Army
-    - Air Force
-    - RAF
-    - Navy
-    - Military
 ---
 
-## Postgraduate bursaries and scholarships
-
-Postgraduate teaching bursaries and scholarships are only available for the subjects listed below. You cannot receive both a teaching bursary and a scholarship.
-
-You must be eligible for student finance to receive a bursary or scholarship (unless you’re training to teach physics or languages).
-
-You cannot receive a scholarship or bursary if you're on a [salaried teacher training course](/funding-and-support/salaried-teacher-training).
-
-| Subject                                                               | Bursary  | Scholarship      |
-|-----------------------------------------------------------------------| -------: | ---------------: |
-| **Art and design**                                                    | $bursaries_postgraduate_artanddesign$  |                  |
-| **Biology**                                                           | $bursaries_postgraduate_biology$  |                  |
-| **Chemistry**                                                         | $bursaries_postgraduate_chemistry$  | $scholarships_chemistry$          |
-| **Computing**                                                         | $bursaries_postgraduate_computing$  | $scholarships_computing$          |
-| **Design and technology**                                             | $bursaries_postgraduate_designandtechnology$  |                  |
-| **English**                                                           | $bursaries_postgraduate_english$  |                  |
-| **Geography**                                                         | $bursaries_postgraduate_geography$  |                  |
-| **Languages**<br/> (French, German and Spanish only)                  | $bursaries_postgraduate_languagesfrenchgermanspanish$  | $scholarships_languagesfrenchgermanspanish$      |
-| **Languages**<br/> (all other languages, including ancient languages) | $bursaries_postgraduate_languagesother$  |                  |
-| **Maths**                                                             | $bursaries_postgraduate_maths$  | $scholarships_maths$         |
-| **Music**                                                             | $bursaries_postgraduate_music$  |                  |
-| **Physics**                                                           | $bursaries_postgraduate_physics$  | $scholarships_physics$        |
-| **Religious education**                                               | $bursaries_postgraduate_religiouseducation$  |                  |
-
-Table caption: Available postgraduate bursaries and scholarships
-
-### Bursaries
+## Eligibility for bursaries
```

Comment:
Currently we say `You must be eligible for student finance to receive a bursary or scholarship (unless you’re training to teach physics or languages).`

Is this no longer the case? Worth checking with policy